.travis.yml

```diff
@@ -3,18 +3,19 @@ dist: xenial
 sudo: false
 before_install:
 - gem install bundler
+- curl --fail --retry 3 --retry-delay 1 --connect-timeout 3 --max-time 30 https://cli-assets.heroku.com/install.sh | sh
 - composer self-update
 install:
 - bundle install
 before_script:
 - bundle exec hatchet ci:setup
-- heroku update
-script: bundle exec parallel_rspec -n 7 --group-by runtime --unknown-runtime 1 --runtime-log "test/var/log/parallel_runtime_rspec.${STACK}.log" --verbose-process-command --combine-stderr --prefix-output-with-test-env-number test/spec/
+script: bundle exec parallel_rspec -n 16 --group-by runtime --unknown-runtime 1 --runtime-log "test/var/log/parallel_runtime_rspec.${STACK}.log" --verbose-process-command --combine-stderr --prefix-output-with-test-env-number test/spec/
 after_script:
 - bundle exec hatchet destroy --all
 - cat parallel_runtime_rspec.log | grep -E '^test/spec/[a-z0-9_/\.-]+\.rb:[0-9]+\.[0-9]+$' | sort
 env:
   global:
+  - HEROKU_DISABLE_AUTOUPDATE=1
   - HATCHET_RETRIES=3
```

Comment:
Hmh, maybe. @schneems ?

Reply:
Since I started tracking flappy runs, I've already caught and squashed a number of issues even with the retry. You can see the tracking issue https://github.com/heroku/hatchet/issues/120.

My ultimate goal with reducing flappiness is to help with productivity. It's frustrating to make a tiny doc commit or version bump, wait for tests...only to have them fail and need to be restarted. If you want to drop it temporarily on your repo for maybe a few weeks or a month, then feel free to go for it, but there are things like network glitches and other general distributed computing concerns that can never be truly sussed out and retries can help there. 

I guess I would say: Go for it if you want. Know that I don't think it's a reasonable long term goal to disable all retries. I wouldn't want to mandate we reduce this value across the team (at least for now).

Reply:
Yeah I wasn't saying we should mandate it. It was just that https://salesforce.quip.com/B8a9A6ZP6APx was scoring us based on how many intermittent tests we've reported - and I've not reported many not because I haven't gotten around to it, but because I haven't seen many. I wondered if the retries might be the reason why :-)

Reply:
If you've got the energy for it, I would be interested to see how much of a difference it makes. It's a sound theory and would be interesting to share the results with the program after a period of trying it.